examples/plot_lasso_cv.py

```diff
@@ -0,0 +1,45 @@
+"""
+=======================================================
+Run LassoCV for cross-validation on Leukemia dataset
+=======================================================
+
+The example runs the LassoCV scikit-learn like estimator
+using the Celer algorithm.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from sklearn.datasets import fetch_mldata
+
+from celer import LassoCV
```

Comment:
I don't get the same figure if I replace this line with:

from sklearn.linear_model import LassoCV

Reply:
Probably fit intercept defaulting to True. I'll fix it and the non ascii characters when I land